library/src/main/kotlin/com/chuckerteam/chucker/api/Chucker.kt

```diff
@@ -75,6 +81,43 @@ public object Chucker {
         NotificationHelper(context).dismissNotifications()
     }
 
+    /**
+     * Export the chucker transactions to a file.
+     *
+     * This must be called on a background thread.
+     *
+     * @param context Application context
+     * @param maxTransactions Maximum number of transactions to return in the file. Passing null
+     * means no limit on the number of transactions
```

Comment:
Why would want to limit this? I don't fully understand the use case.

Reply:
I'm not sure I have a specific use case but it seems like a good idea to have a limit call on an API like this.

Dumping the whole list of transactions from the database can take considerable time, so having ways to limit it and make this API a bit quicker would be good imo.

With the API like this users can specify either a time limit (e.g. calculate the last 1 hour) or can set a count limit (e.g. show the last 100 HTTP messages)

Reply:
I would not add them, till we have a valid use case. We can still add them afterwards if needed. Having the method as suspending allows it to run on a separate thread, so having it as a long operation is not a major deal imho.

Between the two parameters, the timestamp seems more reasonable. The limit instead, will be useless if not paired with a proper pagination support (as you'll be essentially losing some transactions).